charts/ndm/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: openebs-ndm
+description: Disk inventory and management in Kubernetes cluster
```

Comment:
this should be `Kubernetes Storage Device Management. Also used to maintain the inventory of block devices.`